htmd/parameterization/dihedral.py

```diff
@@ -236,32 +195,25 @@ def _objective(self, x, _):
 
         return rmsd
 
-    def _paramsToVector(self, params):
+    def _paramsToVector(self, params, keys):
         """
         Convert the parameter objects to a vector.
         """
+        vector = []
+        for k in keys:
+            assert len(params.dihedral_types[k]) == self.MAX_DIHEDRAL_MULTIPLICITY
+            for term in params.dihedral_types[k]:
+                vector.append(term.phi_k)
+        for k in keys:
+            for term in params.dihedral_types[k]:
+                vector.append(term.phase)
+        for i in range(self.numDihedrals):
```

Comment:
Why not just
```python
vector += [0] * self.numDihedrals
```

Reply:
I was making the other two loops already so I thought I might as well make the third explicit